packages/react-native-win32/.flowconfig

```diff
@@ -115,34 +113,10 @@ module.file_ext=.js
 module.file_ext=.json
 module.file_ext=.win32.js
 
-module.system=haste
```

Comment:
>system [](start = 7, length = 6)

We should have gotten rid of this in the 61 timeframe, since we no longer use Haste and this can lead to flow following the wrong requires